bundles/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/handler/SoulissGenericHandler.java

```diff
@@ -90,7 +90,6 @@ public void initialize() {
                         "N" + Integer.toString(iNode) + "S" + Integer.toString(iSlot));
             }
         } catch (Exception e) {
-            logger.warn("Error getting node/slot from souliss typical (thing config).");
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "configuration error");
```

Comment:
You could make this error message more helpful by replacing it by the message you deleted.